docs/Contributing.md

```diff
@@ -0,0 +1 @@
+random notes about contributing can go here
```

Comment:
Could you move this up a directory and make it `CONTRIBUTING.md`? That way GH can pick it up and display it under issues and PRs.